src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -25,40 +26,50 @@ const DomainList = (props) => {
     isHashtag,
     handleSetFollow,
     handleSetUnFollow,
+    handleSetBlock,
+    handleSetUnblock,
     onPressBody,
     DefaultImage,
-    isCommunity
+    isCommunity,
+    isBlockedSection
   } = props;
 
   return (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new props `handleSetBlock`, `handleSetUnblock`, and `isBlockedSection` have been added. Ensure these are being passed correctly from the parent component and that they are handling their respective functionalities as expected.



<!-- This is an auto-generated comment by OSS CodeRabbit -->